_config.yml

```
@@ -28,6 +28,8 @@ aux_links:
     - /contribute
   source:
     - https://github.com/endoflife-date/endoflife.date
+  donate:
+    - https://opencollective.com/endoflife-date/donate
```

The GitHub link looks much better, so let's use that instead?

```suggestion
    - https://github.com/sponsors/endoflife-date
```